### DISCUSSION TITLE
Use images from Contentful at run-time with gatsby-plugin-image

### DISCUSSION THREAD

**Discussion by davidpaulsson** [gatsbyjs/gatsby#35838] [Help] [answered]
`gatsby-source-shopify` has an `gatsby-plugin-image` helper that allows you to use run-time images, like this

```
const image = storefrontProduct.images.edges[0].node;
const imageData = getShopifyImage({
  image,
  width: 200,
  height: 200,
  layout: "fixed",
});
```

Is there something similar available for `gatsby-source-contentful`?

**Comment by LekoArts** [comment, marked as answer]
Do you mean https://github.com/gatsbyjs/gatsby/blob/release-notes-4.16/docs/docs/reference/release-notes/v4.16/index.md#usecontentfulimage-hook ?

**Reply by davidpaulsson** [reply]
Haha, yes this is EXACTLY what I mean :) I must have been blind. Thank you @LekoArts
